src/app/components/PatientSummaryViewer/resources/Composition/index.tsx

```
@@ -520,6 +522,57 @@
+                  border: '1px solid gray',
+                  padding: '8px',
+                  textAlign: 'left',
+                },
+              },
+            }}
+          >
+            <StyledButton size="small" variant="contained" onClick={toggleInfo}>
+              Toggle Generated
+            </StyledButton>
+            <div
+              dangerouslySetInnerHTML={{ __html: text.div }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)